component/attr_cache/attr_cache.go

```diff
@@ -485,8 +493,10 @@ func (ac *AttrCache) GetAttr(options internal.GetAttrOptions) (*internal.ObjAttr
 
 	if err == nil {
 		// Retrieved attributes so cache them
-		if len(ac.cacheMap) < maxTotalFiles {
+		if len(ac.cacheMap) < int(ac.maxTotalFiles) {
```

Comment:
If we declare the variable as int itself, we dont need to do this conversion for every call